README.md

```diff
@@ -88,6 +89,15 @@ This will start the process of "ejecting" from Create React Native App's build s
 * **One Build Tool**: If you just want to get started with React Native, you shouldn't need to install Xcode, Android Studio, NDKs, or mess with environment variables.
 * **No Lock-In**: You can always "eject" to your own build setup if you need to write custom native code or modify how your app is built.
 
+## Known Issues
+
+There are some issues that have been identified by us and are documented here:
+
+* **QR code can't be scanned when using Webstorm's built-in terminal** [#49](https://github.com/react-community/create-react-native-app/issues/49)
```

Comment:
WebStorm has configurable terminal color themes, I think we want to be specific that this issue appears to be attached to certain themes rather than all of Webstorm.